com.unity.render-pipelines.universal/Documentation~/writing-shaders-urp-unlit-color.md

```diff
@@ -0,0 +1,110 @@
+# URP unlit shader with color input
+
+The shader in this example adds the __Base Color__ property to the Material. You can select the color using that property and the shader fills the mesh shape with the color.
+
+Use the ShaderLab code from section [URP unlit basic shader](writing-shaders-urp-basic-unlit-structure.md) and make the following changes to it:
+
+1. Add the `_BaseColor` property definition to the Properties block:
+    
+    ```c++
+    Properties
+    { 
+        _BaseColor("Base Color", Color) = (1, 1, 1, 1)
```

Comment:
I think maybe having info about the [MainColor] and [MainTexture] would be smart here. @eh-unity can give you more info on how that works.

Reply:
This is a great suggestion, but requires editing a page in the main Unity Manual, or maybe creating a new one for URP. This is a task for the next iteration.
I've created a ticket for this:
https://unity3d.atlassian.net/browse/DOC-2084